manage-data/lifecycle/data-stream/tutorial-data-stream-retention.md

```diff
@@ -10,7 +10,7 @@ products:
 
 # Tutorial: Configure data stream retention [tutorial-manage-data-stream-retention]
 
-In this tutorial, we are going to go over the data stream lifecycle retention; we will define it, go over how it can be configured and how it can gets applied. Keep in mind, the following options apply only to data streams that are managed by the data stream lifecycle.
+In this tutorial, we are going to go over the data stream lifecycle retention; we will define it, go over how it can be configured and how it gets applied. Keep in mind, the following options apply only to data streams that are managed by the data stream lifecycle.
```

Comment:
```suggestion
This tutorial demonstrates lifecycle retention, showing how to define, configure, and apply it. Keep in mind that the following options apply only to data streams that are managed by data stream lifecycle.
```

I might rephrase this a bit to avoid the "royal we" and also to avoid the duplication of "go over".